.idea/codeStyles/codeStyleConfig.xml

```diff
@@ -0,0 +1,5 @@
+<component name="ProjectCodeStyleConfiguration">
```

Comment:
I'm sure thoughts/opinions are all over on this, but I prefer to `.gitignore` all of `.idea`

Reply:
^^

Reply:
done!